src/main/java/bio/terra/cli/command/notebooks/Create.java

```diff
@@ -102,6 +109,10 @@ protected void execute() {
     // each zone.
     envVars.put("SUBNET", "projects/" + projectId + "/regions/" + zone + "/subnetworks/subnetwork");
     envVars.put("TERRA_WORKSPACE_ID", workspaceContext.getWorkspaceId().toString());
+    if (postStartupScript != null) {
```

Comment:
What happens if the url/path is invalid or not accessible? Just thinking that if it's an obscure error message, would it be worth checking that we can access that script here?

Reply:
The notebook instance will be created successfully and the script will silently fail to run.
While right now the pet SA is used on the VM (and its credentials are used to fetch to postStartupScript), when we switch to WM, it will be a new SA on the VM. So we shortly won't be able to validate for the user that their script is accessible by a SA that doesn't exist yet.
I'm inclined to leave as-is for now, and we'll keep an eye on the usability of this.